_topic_map.yml

```diff
@@ -726,10 +726,10 @@ Topics:
   File: setting-up-trusted-ca
   Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
 ---
-#Name: OpenShift Pipelines
-#Dir: openshift_pipelines
-#Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#Topics:
+Name: OpenShift Pipelines
```

Comment:
Shouldn't the bucket name be just _Pipelines_? It doesn't look consistent in the nav:
<img width="271" alt="Screenshot 2020-04-29 at 08 47 07" src="https://user-images.githubusercontent.com/1174934/80597584-09f17280-89f6-11ea-90a2-23dc3f0618f4.png">

Reply:
adding @Preeticp

Reply:
@siamaksade the name for the bucket was fixed based on our initial meetings with Ali, Brandi, Joan wrt [integrating pipeline docs with OCP](https://docs.google.com/document/d/1mqS_GhUbMtZUmtrmGvHveWU-wq0P3IfMVLZifFxtFKk/edit#heading=h.2nmh43mt27ix) and was added almost a year back.

Changing the bucket name for consistency is doable but I would like to clarify one thing - do we want to change only the bucket name? Or we want to change the product name too? Changing the product name would mean changing the mention of OpenShift Pipelines throughout our docs to Pipelines.
For instance, Serverless does it this way: https://docs.openshift.com/container-platform/4.3/serverless/serverless-getting-started.html
How do we want to go about it?
Another reason for calling ourselves OpenShift Pipelines was  that we wanted to distinguish ourselves from the existing Pipeline build, see the note here: https://docs.openshift.com/container-platform/4.3/builds/understanding-image-builds.html#builds-strategy-pipeline-build_understanding-image-builds

Reply:
@preethit just the bucket name. Product name remains OpenShift Pipelines, but in the bucket it looks better IMO to just show "Pipelines" instead of product name.